src/index.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+function Modal(clickElm, modalParent, options) {
```

Comment:
I think it would be nice to name this ``PaymentClient`` instead of ``Modal`` because that seems to be a logical way to use the module